artiq/protocols/asyncio_server.py

```diff
@@ -44,6 +44,8 @@ def __init__(self):
 
     def _client_done(self, task):
         self._client_tasks.remove(task)
+        if task.exception() and task.exception() != asyncio.CancelledError:
+            raise task.exception()
```

Comment:
What handles that exception then?
BTW isn't there a asyncio debug mode that already shows those tasks terminating with exceptions? Do we need additional code?

Reply:
Nothing by default handles this, but the exception deterministically raises here instead of whenever the garbage collector cleans up the task, which varies

Reply:
Maybe logging.error would be more appropriate then?

Reply:
Because I don't see any way that this exception could get caught.

Reply:
There's not currently a logger available in `asyncio_server`. I could add it